backend/controllers/attendanceStatistics.js

```diff
@@ -0,0 +1,106 @@
+const moment = require('moment')
+const { merge, switchDateFormat, getCurrentShift, pascalToString } = require('../utils')
```

Comment:
does this new page work for you?  I get `"log is not a function"` when clicking the link